pkg/render/logstorage.go

```diff
@@ -91,6 +91,7 @@ const (
 	ElasticsearchInternalPort       = 9300
 	ElasticsearchOperatorUserSecret = "tigera-ee-operator-elasticsearch-access"
 	ElasticsearchAdminUserSecret    = "tigera-secure-es-elastic-user"
+	ElasticsearchLinseedUserSecret  = "tigera-ee-linseed-elasticsearch-access-gateway"
```

Comment:
I don't think this var name needs to end with `gateway` - Linseed shouldn't be going via es-gateway.

Reply:
Fixed in my newest enterprise change. Added a "DirectConnection" field to the `User` struct that when set to true will result in the suffix `user-secret` instead of `access-gateway`.